workflow-ui/internal-testing-android/src/test/java/com/squareup/workflow1/ui/internal/test/IdlingDispatcherTest.kt

```diff
@@ -11,9 +11,12 @@ import org.junit.Assert.fail
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
 import kotlin.coroutines.ContinuationInterceptor
 import kotlin.coroutines.CoroutineContext
 
+// The SDK should be unimportant, but Robolectric 4.6.1 has shadowing issues if it isn't set.
```

Comment:
Do you want to bump the Robolectric version while you're here?

Reply:
I wasn't sure if it would wind up breaking anything else.  I can give it a shot.